_docs/docker.md

```diff
@@ -221,7 +221,7 @@ When the **docassemble** container starts, it runs one command:
 /usr/bin/supervisord -n -c /etc/supervisor/supervisord.conf
 {% endhighlight %}
 
-(This is specified in the [Dockerfile], if you are curious.)
+(This is specified in the [Dockerfile][`docassemble/Dockerfile`], if you are curious.)
```

Comment:
Previously this just appeared as the literal text `[Dockerfile]` (i.e., unhyperlinked, with the brackets around it) because it wasn't specified at the bottom of the document. Here I've converted it to a reference-style link, so that it uses the same URL as the other link to the Dockerfile elsewhere in this document.